EPRN.Waste.API/Repositories/Interfaces/IRepository.cs

```diff
@@ -48,5 +50,7 @@ public interface IRepository
         Task<bool> Exists(int journeyId);
 
         Task<string> GetWasteNote(int journeyId);
+
+        Task<Category?> GetCategory(int journeyId);
```

Comment:
I believe all waste journeys have the category set as soon as they are created, therefore Category does not need to be nullable. Please check and change if i am correct

Reply:
Done.